index.js

```diff
@@ -26,16 +25,32 @@ export {
 };
 
 let loaded;
-export async function loadOnce(mediatorUrl = DEFAULT_MEDIATOR) {
+export async function loadOnce(options) {
   if(loaded) {
     return loaded;
   }
 
   loaded = true;
-  return load(mediatorUrl);
+  return load(options);
 }
 
-export async function load(mediatorUrl = DEFAULT_MEDIATOR) {
+export async function load(options = {
+  mediatorOrigin: DEFAULT_MEDIATOR_ORIGIN
+}) {
+  // backwards compatibility (`options` used to be a string for expressing
```

Comment:
Would it be too disruptive to just drop old support and do a breaking release?

Reply:
Yeah, in this release and at this time. We can do a breaking one later if we feel we need to.